packages/components/src/MaskedInput.tsx

```diff
@@ -472,6 +472,11 @@ MaskedInput.defaultProps = {
     // no-op
   },
   getNextSegmentValue: (range, delta, segmentValue) => segmentValue,
+  selection: {
```

Comment:
Why was a default prop needed?

Random side note, was reading this in the TypeScript cheat sheet: https://github.com/typescript-cheatsheets/react/blob/main/README.md#you-may-not-need-defaultprops

Reply:
The linter plugin has a false positive with forwardRef components

https://github.com/yannickcr/eslint-plugin-react/issues/2856